tests/foreman/cli/test_repository.py

```diff
@@ -361,8 +523,15 @@ def test_positive_update_url(self):
         @Assert: Repository url is updated
         """
         new_repo = self._make_repository()
+        # generate a tuple of auth repos with all valid credentials
+        auth_repos = ()
```

Comment:
@oshtaier you realize that the loop is there just for constructing the final string, right? Perhaps putting it into `setup` should make it more clear